libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -436,14 +436,18 @@ class AP_AHRS
         return _home;
     }
 
-    enum HomeState home_status(void) const {
-        return _home_status;
+    // functions to handle locking of home.  Some vehicles use this to
+    // allow GCS to lock in a home location.
+    void set_home_locked(bool locked) {
```

Comment:
Any reason to not call this `lock_home(void)`? It seems we only want to lock, never unlock.

Reply:
Eh.  Flexibility.  Doesn't hurt.